nntrainer/graph/network_graph.cpp

```diff
@@ -386,7 +421,16 @@ int NetworkGraph::addLossLayer(const LossType loss_type) {
   last_node = Sorted.back();
   std::string input_str = last_node.layer->getName();
 
-  std::shared_ptr<LossLayer> layer = std::make_unique<LossLayer>();
+  std::shared_ptr<Layer> layer = nntrainer::createLayer(LossLayer::type);
+
+  status =
+    std::dynamic_pointer_cast<LossLayer>(layer)->setLoss(updated_loss_type);
```

Comment:
Later, I think we should merge this to setProperty as well. Could you put todo here?

Reply:
Why do you think we need to merge this into setProperty?? It is Propery from internal loss layer. Not dist layer.

Reply:
loss layer has to be sooner or later open as an interface, so it would be better to use the existing interface if it's possible.

Reply:
yes. loss layer might be open. but time dist layer is not. and I'm not sure if we have to setLoss should be opened.

Reply:
I think this can be done as sth like `layer->setProperty("loss=loss_cross_entropy")` later :)